doc/api/v0.yaml

```diff
@@ -337,6 +337,71 @@ paths:
                       }
                     ]
 
+  /api/v0/fragment/statuses:
+    post:
+      description: Get statuses of fragments
+      operationId: FragmentStatuses
+      tags:
+        - fragment
+      requestBody:
+        description: List of fragment IDs
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                type: string
+                pattern: '[0-9a-f]+'
+            example: ["68dcc12fe0dfe5e7b66ca6f8c959f9aa43b273e120a77fc3e4e2f04f1ecd7968"]
+          
+      responses:
+        '200':
```

Comment:
what if a fragment is not in the log (GCed or ...) ? What will the user gets?

Reply:
For now here is no 404, so the value will be just missing from the dictionary. Not very good behavior, but I didn't find a way to make fast fragment lookup in the blockchain.

Reply:
document that if the fragment is no longer in the log it does not mean it was not accepted or accepted and that it is now better to query the explorer API